### PR TITLE
Update ButtonTemplateBuilder.php

### PR DESCRIPTION
**Pull request by cyb3rwin**
Make thumbnailImageUrl optional, so user can send empty imageurl.
Ref: https://developers.line.me/en/docs/messaging-api/reference/#buttons

### CODE REVIEW

File: src/LINEBot/MessageBuilder/TemplateBuilder/ButtonTemplateBuilder.php [hunk 1]
```diff
@@ -75,12 +75,16 @@ public function buildTemplate()
...
             'type' => TemplateType::BUTTONS,
-            'thumbnailImageUrl' => $this->thumbnailImageUrl,
             'title' => $this->title,
             'text' => $this->text,
             'actions' => $actions,
         ];
-
+        
+        /* Make thumbnailImageUrl optional */
+        if(!empty($this->thumbnailImageUrl)){
```
Comment (Vaduz) [inline src/LINEBot/MessageBuilder/TemplateBuilder/ButtonTemplateBuilder.php]: Thank you for opening p-r.

The thumbnailImageUrl is string, so please write like
```php
if ($this->thumbnailImageUrl) {
```
I found inconsistent with the official document, the `title` is also optional.
Can I optionalize it incidentally?

I think you need to change constructor as well like this:
```php
public function __construct($title = null, $text, $thumbnailImageUrl = null, array $actionBuilders)
```
This constructor looks ugly though :(

Plus you need to add test cases that you have changed.

**Comment by cyb3rwin** [comment]
Test case https://github.com/line/line-bot-sdk-php/pull/152

**Comment by Vaduz** [comment]
Please include tests to this p-r

**Comment by Vaduz** [comment]
This is fixed by #174

**Pull request closed without merging**